pkg/sys/mount/discovery.go

```diff
@@ -14,9 +16,9 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-package sys
+package mount
```

Comment:
Why do you need a new package?

Reply:
moved all the mount related functions to separate pkg as suggested by @wlan0